File: JS/Tema3_Ejercicio1.js
/*
 * TEMA 3: ESTRUCTURAS CONDICIONALES - EJERCICIO 1
 * ================================================
 * Comparar dos números e indicar cuál es mayor.
 * 
 * EXPLICACIÓN:
 * - Sentencias if-else-if
 * - Operadores de comparación: <, >, ==
 * - Igualdad vs desigualdad
 */

function compararNumeros(a, b) {
    if (a > b) {
        return `${a} es mayor que ${b}`;
    } else if (a < b) {
        return `${a} es menor que ${b}`;
    } else {
        return `${a} es igual a ${b}`;
    }
}

console.log("=== COMPARACIÓN DE NÚMEROS ===");
console.log(compararNumeros(15, 10));
console.log(compararNumeros(5, 20));
console.log(compararNumeros(42, 42));

/*
 * SALIDA ESPERADA:
 * === COMPARACIÓN DE NÚMEROS ===
 * 15 es mayor que 10
 * 5 es menor que 20
 * 42 es igual a 42
 */

File: Java/Tema3_Ejercicio1.java
/*
 * TEMA 3: ESTRUCTURAS CONDICIONALES - EJERCICIO 1
 * =================================================
 * Escriba un programa que lea dos números y determine cuál es el mayor.
 */

import java.util.Scanner;

public class Tema3_Ejercicio1 {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        
        System.out.print("Digite el primer número: ");
        int numero1 = scanner.nextInt();
        
        System.out.print("Digite el segundo número: ");
        int numero2 = scanner.nextInt();
        
        System.out.println("\n=== COMPARACIÓN DE NÚMEROS ===");
        
        if (numero1 == numero2) {
            System.out.println("Los números son iguales.");
        } else if (numero1 > numero2) {
            System.out.println("El mayor es: " + numero1);
        } else {
            System.out.println("El mayor es: " + numero2);
        }
        
        scanner.close();
    }
}

/*
 * SALIDA ESPERADA:
 * Digite el primer número: 10
 * Digite el segundo número: 20
 * 
 * === COMPARACIÓN DE NÚMEROS ===
 * El mayor es: 20
 */
